### PR TITLE
[10.0][ADD] Partner Alias

### PR DESCRIPTION
**Pull request by woodbrettm**
Hi all,

This module adds the concept of firstname aliases to partners. The aliases can be found in the Personal Information Tab in the partner form.

Thanks!
Brett
@lasley @tedsalmon

### CODE REVIEW

**Comment by woodbrettm** [comment]
Build is failing from:
* [partner_contact_birthdate](https://travis-ci.org/OCA/partner-contact/jobs/196859100#L742)
* @lasley I need to add security csv file, but not quite sure which groups to add to the file.

File: partner_alias/views/res_partner_alias_view.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.partner.alias</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="search_view_id" ref="res_partner_alias_view_search" />
+        <field name="domain">[]</field>
+        <field name="context">{}</field>
+    </record>
+
+</odoo>
```
Comment (woodbrettm) [inline partner_alias/views/res_partner_alias_view.xml]: Newline

**Comment by lasley** [comment]
Use `base.group_user` for the security. I don't think the warning is failing you. from the other module.

**Comment by woodbrettm** [comment]
Build is passing now.

File: partner_alias/views/res_partner_view.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <field name="name">Personal information page for contacts form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="partner_contact_personal_information_page.personal_information" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='personal_information_page']" position="inside">
+                <group string="Aliases" name="aliases">
+                    <field name="alias_ids" nolabel="1">
+                        <tree editable="bottom"
+                              context="{'default_partner_id': id}"
+                              >
```
Comment (tedsalmon) [inline partner_alias/views/res_partner_view.xml]: Any reason you put the close tag out here all alone? Looks kinda weird to me, but I'm not sure if you were going for something in particular :)

File: partner_alias/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+res_partner_alias_user,res_partner_alias group_user,model_res_partner_alias,base.group_user,1,1,1,1
```
Comment (tedsalmon) [inline partner_alias/security/ir.model.access.csv]: No newline at EOF :(

**Review by tedsalmon — APPROVED** [review]
LGTM, minus a couple of small issues! 👍

**Comment by woodbrettm** [comment]
Thanks @tedsalmon fixed!

**Comment by woodbrettm** [comment]
@lasley ping

**Review by lasley — APPROVED** [review]
Thanks @bmw95

**Review by elicoidal — APPROVED** [review]
spotless

**Pull request merged**